### ISSUE TITLE
xref not working?

### ISSUE THREAD

**Issue by mowangjuanzi** [php/doc-en#1415]
source: https://github.com/php/doc-en/blob/master/reference/com/functions/com-load-typelib.xml

```xml
<para>
   Note that it is much more efficient to use the <xref
   linkend="ini.com.typelib-file"/> configuration setting to pre-load and
   register the constants, although not so flexible.
  </para>
```

php.net: https://www.php.net/manual/en/function.com-load-typelib

```text
Note that it is much more efficient to use the configuration setting to pre-load and register the constants, although not so flexible.
```

and `<xref linkend="ini.com.autoregister-typelib"/>`

Don't know if there are others? I just saw this one

**Comment by cmb69** [comment]
Oh, right! `xref` only works, if there is a proper description at the link-end. Otherwise the link text needs to be written manually.

**Comment by cmb69** [comment]
Ah, thank you! Fixed as 554db5a3a3ac77910d68349eefa37e2c9a9da628.
